parser/gen/CEL.g4

```diff
@@ -78,9 +78,9 @@ mapInitializerList
     ;
 
 literal
-    : tok=NUM_INT   # Int
+    : sign=MINUS? tok=NUM_INT   # Int
```

Comment:
Remember to update the lexis in cel-spec.

Reply:
Thanks, will do!